docs/ExposePortsToInternet.md

```diff
@@ -0,0 +1,46 @@
+<!--- master only -->
+> ![warning](images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.14.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Expose Container Port to Host-Interface / Internet
+
+In the [Calico without Docker networking tutorial](calico-with-docker/without-docker-networking/README.md) 
+or the [Calico as a Docker network plugin tutorial](calico-with-docker/docker-network-plugin/README.md)
+we created containers and assigned endpoints (a container interface) to them. This is used for Container-
+To-Container communication.
+
+The example below shows how to expose a port of a container to the Host-Interface so this container is 
+reachable from outside / the internet.
+
+## Why isn't the `-p` flag on `docker run` working as expected?
+If you connect containers to the docker0 interface, then calico would not be able to enforce security rules
+between workloads; all containers on the bridge would be able to communicate with each other.
+
+## How can i make a container reachable from the Internet (Host-Interface)?
+Let's say you want to expose port 80 of your container IP 192.168.0.1 to be reachable from the internet. 
+You can do that these two steps:
+
+### Update Profile to allow traffic to container
+First find your profile (see [how to find profile id with docker networking](calico-with-docker/docker-network-plugin/AdvancedPolicy.md))
```

Comment:
This section is for both calico with and without the docker network plugin, but currently it assumes you're using the docker network plugin.

Can you change this to something like:

First, add a rule to your profile to allow inbound TCP traffic to port 80.
> Note: If you are using Calico with the docker networking plugin, you can find your profile ID using the commands from the [Advanced Policy guide](calico-with-docker/docker-network-plugin/AdvancedPolicy.md).